### DISCUSSION TITLE
Grandfathering?

### DISCUSSION THREAD

**Discussion by HPaulson** [deso-protocol/dips#14] [Q&A]
Will already-started large feature changes, such as NFTs, be grandfathered? Is there a list of features we can expect to not have an associated CIP?

As a community we should encourage and ensure the use of CIPs by the core team for all changes, however obviously some changes have already began development, and should continue without one. As such, a list of expected unfinished changes which will not have an associated CIP would be very useful.

**Comment by tijno** [comment]
Suggestion:

low & medium changes already in progress on kanban - no need for full CIP.

large changes in progress - simplified cip focussing on key decisions/variables & features outstanding

any change not in progress on kanban - follow full process.

**Reply by HPaulson** [reply]
Agreed 100%. @maebeam thoughts on this?

**Comment by AikonMishima** [comment]
Was thinking the same in terms of the Coin Locking mechanism.

I tend to agree with @tijno.
Large changes would require CIP input to improve the code in production by the core devs, or at least introduce the community's view on features / potential pitfalls.

All of this, of course, without having an impact on deadlines.
